frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -2,19 +2,37 @@ import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
-  onChange: () => void;
+  checked?: boolean;
+  name?: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
You don't need event as I see from your usage

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/a1e5d5fd-b73b-428b-8a56-3c674e5adc8e)